app/src/components/group_schema_table.rs

```diff
@@ -154,7 +155,35 @@ impl GroupSchemaTable {
         let hardcoded = ctx.props().hardcoded;
         html! {
             <tr key={attribute.name.clone()}>
-                <td>{&attribute.name}</td>
+                {
+                    if hardcoded {
+                        let desc = group::resolve_group_attribute_description_or_default(&attribute.name);
+                        html!{
+                            <td>
+                                {desc.attribute_name.clone()}
+                                {
+                                    if desc.aliases.is_empty() {
+                                        html!{}
+                                    } else {
+                                        html!{
+                                            <>
+                                                <br/>
+                                                <small class="text-muted">
+                                                    {if desc.aliases.len() == 1 {"Alias: "} else {"Aliases: "}}
```

Comment:
Just "Aliases". Let's not add complexity for the sake of proper grammar

Reply:
Check.